ui/src/authentication/state/types.js

```diff
@@ -0,0 +1,6 @@
+export const IS_LOADING = 'fetchUserInfo/IS_LOADING';
```

Comment:
Do `IS_LOADING` and `IS_CONFIRMED_LOADING` have to be different?

Reply:
Yes because otherwise different part of the application are being reload due to redux and I didn't want to create yet another reducer for that :)